src/app/view/components/dot-navigation/components/dot-nav-item/dot-nav-item.component.ts

```diff
@@ -35,6 +40,27 @@ export class DotNavItemComponent {
         });
     }
 
+    /**
+     * Align the submenu top or bottom depending of the browser window
+     *
+     * @memberof DotNavItemComponent
+     */
+    setSubMenuPosition(): void {
+
+        if (this.collapsed) {
+            const [rects] = this.subnav.ul.nativeElement.getClientRects();
+
+            if (rects.bottom > window.innerHeight) {
+                this.customStyles = {
```

Comment:
why not moving this to a CSS class and then based on the IF you add the css class or not

Reply:
Because I want to make sure is not overwritten by any other class.